features/src/main/feature/camel-features.xml

```diff
@@ -944,48 +944,22 @@
-<!--        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.grpc/${google-cloud-grpc-bundle-version}</bundle>-->
-<!--        <bundle dependency='true'>wrap:mvn:com.google.re2j/re2j/${re2j-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-common/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-codec/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-codec-http/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-codec-http2/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-codec-socks/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-resolver/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-transport/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-buffer/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-transport-classes-epoll/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-transport-native-unix-common/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-handler/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-handler-proxy/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>wrap:mvn:io.opencensus/opencensus-proto/${opencensus-proto-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.okhttp/${squareup-okhttp-bundle-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.okio/${squareup-okio-bundle-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.bouncycastle/bcpkix-jdk18on/${bouncycastle-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.bouncycastle/bctls-jdk18on/${bouncycastle-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.bouncycastle/bcprov-jdk18on/${bouncycastle-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.bouncycastle/bcutil-jdk18on/${bouncycastle-version}</bundle>-->
-<!--        <bundle>mvn:org.apache.camel/camel-google-pubsub/${project.version}</bundle>-->
-<!--    </feature>-->
+    <feature name='camel-google-pubsub' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <bundle dependency='true'>wrap:mvn:com.google.cloud/google-cloud-pubsub/1.126.2</bundle>
+        <bundle dependency='true'>wrap:mvn:com.google.api/api-common/2.24.0</bundle>
+        <bundle dependency='true'>mvn:com.google.protobuf/protobuf-java/3.25.2</bundle>
+        <bundle dependency='true'>mvn:com.google.guava/guava/33.0.0-jre</bundle>
```

Comment:
The expected version of guava is `32.1.3-jre` according to the bom of google-cloud-pubsub v 1.126.2 as you can see https://central.sonatype.com/artifact/com.google.cloud/google-cloud-pubsub/1.126.2

Reply:
Yes, the plugin is also telling me the same, but I'm unsure if we should use the same version as in camel: https://github.com/apache/camel/blob/camel-4.4.1/parent/pom.xml#L193

Reply:
also on my side
```
[INFO] +- com.google.cloud:google-cloud-pubsub:jar:1.126.2:compile
[INFO] |  +- io.grpc:grpc-api:jar:1.61.0:compile
[INFO] |  +- io.grpc:grpc-stub:jar:1.61.0:runtime
[INFO] |  +- io.grpc:grpc-protobuf:jar:1.61.0:compile
[INFO] |  +- io.grpc:grpc-protobuf-lite:jar:1.61.0:runtime
[INFO] |  +- com.google.api:api-common:jar:2.24.0:compile
[INFO] |  +- com.google.protobuf:protobuf-java:jar:3.25.2:compile
[INFO] |  +- com.google.api.grpc:proto-google-common-protos:jar:2.32.0:compile
[INFO] |  +- com.google.auth:google-auth-library-oauth2-http:jar:1.22.0:compile
[INFO] |  +- com.google.auth:google-auth-library-credentials:jar:1.22.0:compile
[INFO] |  +- com.google.http-client:google-http-client-gson:jar:1.43.3:compile
[INFO] |  +- com.google.api.grpc:proto-google-cloud-pubsub-v1:jar:1.108.2:compile
[INFO] |  +- com.google.api.grpc:proto-google-iam-v1:jar:1.27.0:compile
[INFO] |  +- com.google.guava:guava:jar:33.0.0-jre:compile
```